### PR TITLE
fix: 🐞 removal of opencv-python-headless package

### PR DESCRIPTION
**Pull request by onuralpszr**
# Description

Hello, 👋 ,  After long testing,user experiences and problems, we decided to remove opencv-python-headless package from supervision and using full version opencv, it is easy swap headless version If user desired to or wants to use distro version of opencv if wished to, we had so many cases bring more problem than gain and I think around 20mb~ bigger package won't be a big problem as well. Thank you so much 

Related 
#1522 
#351 
#556

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -52,13 +52,11 @@ matplotlib = ">=3.6.0"
 pyyaml = ">=5.3"
 defusedxml = "^0.7.1"
 pillow = ">=9.4"
-opencv-python = { version = ">=4.5.5.64", optional = true }
-opencv-python-headless = ">=4.5.5.64"
 requests = { version = ">=2.26.0,<=2.32.3", optional = true }
 tqdm = { version = ">=4.62.3,<=4.66.5", optional = true }
-# pandas: picked lowest major version that supports Python 3.8
```
Comment (LinasKo) [inline pyproject.toml:59]: I'd rather keep this, as it's useful to know why certain dependency versions were chosen.

**Review by LinasKo — CHANGES_REQUESTED** [review]

**Review by LinasKo — CHANGES_REQUESTED** [review]
I believe the lockfile needs to be updated as well.

```bash
pyproject.toml changed significantly since poetry.lock was last generated. Run `poetry lock [--no-update]` to fix the lock file.
```

**Comment by onuralpszr** [comment]
@LinasKo tests are done, index.md corrected (with some small ui fix), merge complete, lock file done as well.

**Comment by LinasKo** [comment]
Merging. Thank you @onuralpszr!

**Comment by onuralpszr** [comment]
docs preview

image

**Pull request merged**